molecule/testinfra/staging/app/test_tor_hidden_services.py

```diff
@@ -2,6 +2,7 @@
 import re
 
 
+testinfra_hosts = ["app-staging"]
```

Comment:
I would ideally love this to be an environment variable. I foresee the host names changing in the future but this isnt a deal-breaker for me. Just noting it for future implementations.

Reply:
That's true, for example if/when we go back to testing e.g. `app-prod` regularly, we'll want to override here. Appending to the list seems sane. Willing to defer, since we don't have a clear picture of the future testing scenarios just yet.